src/containers/language/language-component.js

```diff
@@ -18,7 +18,7 @@ const LanguageComponent = ({ fromSideBar }) => {
   };
 
   useEffect(() => {
-    i18n.changeLanguage(language.current);
+    i18n.changeLanguage(language);
   }, [i18n]);
```

Comment:
maybe you need to add the language to the dependency array
check consolo warnings

Reply:
You are right. I got kind of warning and I fixed it.